packages/teams-js/src/internal/utils.ts

```diff
@@ -504,3 +504,46 @@ export function validateUuid(id: string | undefined | null): void {
     throw new Error('id must be a valid UUID');
   }
 }
+
+/**
+ * @beta
```

Comment:
Probably no need for a @beta tag here since this isn't exported outside of the library

Reply:
removed!